rsconf/component/raydata_scan_monitor.py

```diff
@@ -26,14 +26,29 @@ def internal_build_compile(self):
         z.intake_d = jc.systemd.run_d.join(_INTAKE_D)
 
     def internal_build_write(self):
+        def _env_ok(item):
```

Comment:
I think this is probably not the best way to do this. Rather call pkconfig.to_environ directly:

`pkconfig.to_environ('*', PKDict(pykern=jc.pykern, sirepo=PKDict(raydata=jc.sirepo.raydata)))`

This probably should be a modularized as sirepo_unit_env does something similar. It could be a function in rsconf.component.T called something like python_service_env that would also except an exclude_re so that sirepo_unit_env could use it.

python_service_env would add `PYTHONUNBUFFERED=1`.

sirepo_unit_env should exclude sirepo.raydata.

Reply:
4267ad6d571e91f709cadce4c3cd5ca8f6d0f58f